src/components/AssetScanner/stories/AssetScanner.stories.tsx

```diff
@@ -188,4 +188,80 @@ storiesOf('AssetScanner/Examples', module)
         />
       )}
     />
-  ));
+  ))
+  .add(
+    'Crop placeholder',
+    () => (
+      <AssetScanner
+        onError={onError}
+        ocrRequest={ocrRequest}
+        onImageRecognizeFinish={onImageRecognizeFinish}
+        cropSize={{ width: 400, height: 200 }}
+      />
+    ),
+    {
+      readme: {
+        content: ocrRequestDoc,
```

Comment:
You need to provide a proper description for each story (with code examples and function description if it's needed). For now, for each story in storybook that you've added, you have similar code example (like in custom ocrRequest story)